### PR TITLE
Clarify contribution process

### PR DESCRIPTION
**Pull request by p3r7**
This is an attempt at clarifying the chronology of the contribution process.

- make it explicit that authors should host their scripts on a public git repo 
- now that we have `;install <git_url>`, I feel like creating a lines thread could occur before declaring it on the maiden community repo
- more explicit explanation of the role of a script inline header documentation
- mention about the wiki and that it can be linked to in the maiden community repo entry

I initially tried to keep the `contribute` & `gather` section separated, but had a hard time given how interleaved they were.
It's certainly due to the "chicken & egg problem" between publishing a script and communicating about it on lines + the wiki (w/ all the X-referencing).

I kinda feel that this whole process might be worth having it's own how-to documentation page separated from this maiden page (with maybe a small flowchart showing which steps are optional) but didn't want to make too many changes at once.

### CODE REVIEW

File: norns/maiden.md [hunk 1]
```diff
@@ -70,38 +70,50 @@ Each entry has an **INSTALL** action, which can be used to install the selected
...
+
+You can optionally declare it on the wiki ([norns.community](/docs/norns/community-scripts/)) and write a small documentation about its usage.
+
+So that anyone can do a 1-click install using maiden maiden, it is encourage to register your script to the [community project repo](https://github.com/monome/norns-community/blob/master/community.json). Please submit a pull request with the following information:
 
 ```json
     {
       "project_name": "NAME",
-      "project_url": "URL",
+      "project_url": "GIT_URL",
```
Comment (ngwese) [inline norns/maiden.md]: while using git to host the script provides the most robust experience it is not a requirement for maiden, URLs to zip archives will work as well. suggest leaving this as is and providing more supporting info if need be.

File: norns/maiden.md [hunk 2]
```diff
@@ -70,38 +70,50 @@ Each entry has an **INSTALL** action, which can be used to install the selected
...
       "author": "NAME",
       "description": "WORDS",
-      "discussion_url": "LINES_LINK",
+      "discussion_url": "LINES_URL",
+      "documentation_url": "DOC_URL",
       "tags": ["TAG", "TAG", "TAG"],
       "origin": "IF_APPLICABLE: lines"
```
Comment (ngwese) [inline norns/maiden.md]: not part of this change but we should remove this property from the docs, it is not used and I’ve removed all references to `origin` in the actual catalog

File: norns/maiden.md [hunk 3]
```diff
@@ -70,38 +70,50 @@ Each entry has an **INSTALL** action, which can be used to install the selected
...
+```
+
+Then, create a new thread under the [norns Library on lines](https://llllllll.co/c/library).
+
+The URL in your script brief documentation should point to the lines Library thread for your script. There's a chicken-egg situation with starting a thread and uploading the project, so you may want to edit and upload your project just after creating a thread.
+{: .label}
+
+You can optionally declare it on the wiki ([norns.community](/docs/norns/community-scripts/)) and write a small documentation about its usage.
+
+So that anyone can do a 1-click install using maiden maiden, it is encourage to register your script to the [community project repo](https://github.com/monome/norns-community/blob/master/community.json). Please submit a pull request with the following information:
```
Comment (ngwese) [inline norns/maiden.md]: we should change the link to the catalog to use `main` instead of `master`

File: norns/maiden.md [hunk 4]
```diff
@@ -70,38 +70,50 @@ Each entry has an **INSTALL** action, which can be used to install the selected
...
 
-### contribute
+### contribute & gather
 
-To add to the [community project repo](https://github.com/monome/norns-community/blob/master/community.json), please submit a pull request with the following information:
+The norns ecosystem was created with community as a focus. The exchange of ideas leads to new ideas.
+
+As such, you are encourage to share your creations with the community.
+
+The first step is to host your script on a public git repository (e.g. on GitHub, GitLab ...).
```
Comment (ngwese) [inline norns/maiden.md]: suggest not being semi-prescriptive here and also mentioning the zip option if that is desirable. historically git was seen as a barrier for entry but maybe that view is no longer held?

**Review by ngwese — COMMENTED** [review]
made a few suggestions, mostly around whether we want to only document the git path for sharing.

**Comment by p3r7** [comment]
Addressed all the comments above.

**Comment by ngwese** [comment]
awesome, thanks! i’ll leave the final call to @dndrks as the primary author for much of the docs.

File: norns/maiden.md [hunk 5]
```diff
@@ -70,38 +70,49 @@ Each entry has an **INSTALL** action, which can be used to install the selected
...
-      "project_name": "NAME",
-      "project_url": "URL",
-      "author": "NAME",
-      "description": "WORDS",
-      "discussion_url": "LINES_LINK",
-      "tags": ["TAG", "TAG", "TAG"],
-      "origin": "IF_APPLICABLE: lines"
-    },
-```
+As such, you are encourage to share your creations with the community.
```
Comment (dndrks) [inline norns/maiden.md]: small typo
> As such, you are encouraged to share your creations with the community.

File: norns/maiden.md [hunk 6]
```diff
@@ -70,38 +70,49 @@ Each entry has an **INSTALL** action, which can be used to install the selected
...
+
+The first step is to host the sources of your script publicly, either as a git repository (e.g. on GitHub, GitLab ...) or web-hosted zip archive.
 
-Be sure you include information at the top of your script to help future users:
+Be sure you include a block of information at the top of your script. It will serve as a brief documentation when launching it from norns.
```
Comment (dndrks) [inline norns/maiden.md]: small thing
>Be sure you include the following block of information at the top of your script. It will serve as a brief documentation when launching it from norns.

File: norns/maiden.md [hunk 7]
```diff
@@ -70,38 +70,49 @@ Each entry has an **INSTALL** action, which can be used to install the selected
...
+-- short script description
+--
+-- short usage instructions
 ```
 
-That last line is a link back to the thread number. There's a chicken-egg situation with starting a thread and uploading the project, so you may want to edit and upload your project just after creating a thread.
+Then, create a new thread under the [norns Library on lines](https://llllllll.co/c/library).
 
-### gather
+The URL in your script brief documentation should point to the lines Library thread for your script. There's a chicken-egg situation with starting a thread and uploading the project, so you may want to edit and upload your project just after creating a thread.
```
Comment (dndrks) [inline norns/maiden.md]: typo?
>The URL in your script's brief documentation should point to the lines Library thread for your script.

File: norns/maiden.md [hunk 8]
```diff
@@ -70,38 +70,49 @@ Each entry has an **INSTALL** action, which can be used to install the selected
...
+{: .label}
 
-While many projects are held in the community repository, it can be hard to engage with a project's creator through GitHub. To facilitate discussion, many projects are also shared through the [norns Library on lines](https://llllllll.co/c/library).
+You can optionally declare it on the wiki ([norns.community](/docs/norns/community-scripts/)) and write a small documentation about its usage.
```
Comment (dndrks) [inline norns/maiden.md]: rephrase suggestion:

> You can optionally declare it on [norns.community](/docs/norns/community-scripts/), a community wiki which provides tools and a platform to create compelling documentation which helps other artists use your script.

File: norns/maiden.md [hunk 9]
```diff
@@ -70,38 +70,49 @@ Each entry has an **INSTALL** action, which can be used to install the selected
...
+
+So that anyone can do a 1-click install using maiden maiden, it is encourage to register your script to the [community project repo](https://github.com/monome/norns-community/blob/main/community.json). Please submit a pull request with the following information:
```
Comment (dndrks) [inline norns/maiden.md]: rephrase suggestion:

> So that anyone can do a one-click install using maiden, we encourage you to register your script in the [community project repo](https://github.com/monome/norns-community/blob/main/community.json). Please submit a pull request with the following information:

**Review by dndrks — CHANGES_REQUESTED** [review]
oh wowow, super rad to have your help on this, thank you @p3r7 !!!

i have only a few super-tiny grammar + phrasing notes. please let me know if you have any q's or feedback!

**Comment by p3r7** [comment]
Thanks for the prompt feedback.

Applied all the fixes + suggestions.

**Review by dndrks — APPROVED** [review]
raaaaad! thank you jordan!!
